images/kubeflow/main.tf

```diff
@@ -0,0 +1,83 @@
+terraform {
+  required_providers {
+    apko = { source = "chainguard-dev/apko" }
+  }
+}
+
+locals {
+  components = toset([
+    "jupyter-web-app",
+    "volumes-web-app",
+  ])
+
+  packages = merge(
+    { for k in local.components : k => k },
+    {
+      "jupyter-web-app" = "kubeflow-jupyter-web-app"
+      "volumes-web-app" = "kubeflow-volumes-web-app"
+    },
+  )
+
+  repositories = merge(
+    { for k in local.components : k => k },
+    {
+      "jupyter-web-app" = "${var.target_repository}-jupyter-web-app"
+      "volumes-web-app" = "${var.target_repository}-volumes-web-app"
+    },
+  )
+}
```

Comment:
I don't think this is right because the README ends up having `cgr.dev/kubeflow` as the image, but if I understand correctly, this file produces `cgr.dev/kubeflow-jupyter-web-app` and `cgr.dev/kubeflow-volumes-web-app`.

My understanding is that we assume each directory under `images/` maps to a single `cgr.dev` repository. I would split this into two `images/kubeflow-jupyter-web-app` and `images/kubeflow-volumes-web-app`.

Reply:
the way it is should be fine, upstream also segments things in a similar way (`kubeflow vs pipelines`).

so our single repo `cgr.dev` assumption just means we end up with flattened repos like so:

```
cgr.dev/chainguard/kubeflow-jupyter-web-app (maps to upstream kubeflow/kubeflow)
cgr.dev/chainguard/kubeflow-pipelines-api-server (maps to upstream kubeflow/pipelines)
```